d3a_interface/device_validator.py

```diff
@@ -60,9 +61,34 @@ def validate_energy(cls, **kwargs):
     @classmethod
     def validate_rate(cls, **kwargs):
         """Validate rates of a Home Meter device."""
+
+        cls._validate_fit_to_limit(
+            fit_to_limit=kwargs.get("fit_to_limit"),
+            energy_rate_increase_per_update=kwargs.get("energy_rate_increase_per_update"),
+            energy_rate_decrease_per_update=kwargs.get("energy_rate_decrease_per_update"))
         cls._validate_home_meter_consumption_rates(**kwargs)
         cls._validate_home_meter_production_rates(**kwargs)
 
+    @staticmethod
```

Comment:
I assume that refactoring this in order to use this method and not the duplicated code that checks fit to limit-related parameters for each device is part of a following PR , right ?

Reply:
Unfortunately, I'm not sure how to proceed with that. This function deals with two energy rates:
- `energy_rate_increase_per_update`
- `energy_rate_decrease_per_update`
Most devices only have one of the two, which means that the other one will always be `None`. For those devices, this would then always raise the exception, which is not what we want.

Maybe I'm missing something though. Do you have any suggestions to re-use this?